### ISSUE TITLE
Custom report resets when applying selections on tables

### ISSUE THREAD

**Issue by Luis-Feilem** [ClimberAB/ClimberCustomReport#27]
Hello there,

This is a **great extension** and **works wonderfully** so far, **_but_** I encountered a fairly annoying issue: when creating the custom report based off a Table Master Item (Table, not Pivot Table), **_every time a selection is made on the table, the custom report resets_**, forcing you to configure the custom report again from start... which can get on your nerves the second time it happens.

Is this a known feat? Am I doing something out of the ordinary maybe?
Thank you for your good work.
Sincerely,
Luis-Feilem.

**Comment by ClimberExtensions** [comment]
Do you have an expression on the visualisation, e.g. in the Title, footer?

**Comment by Luis-Feilem** [comment]
I do have expressions among my measures, just Aggregate functions though, like Min, Max, Sum. Some of them include Set Analysis expressions.

Their contents are identical to some Pivot Tables in which this issues does not occur.

**Comment by ClimberExtensions** [comment]
Could you attach a QVF file where the bug is recreated?
//Karl

**Comment by Luis-Feilem** [comment]
Will do, but it will take me some time to anonymize its contents... Hopefully I will upload it within today.

Cheers

**Comment by Luis-Feilem** [comment]
Sorry for the delay, we got overflowed with work and I couldn't get a couple of hours to make this (priorities...).

Thanks you for your attention and support.
Cheers,
Luis.
[CustomReport_Issue_StraightTableReset.zip](https://github.com/ClimberAB/ClimberCustomReport/files/1058463/CustomReport_Issue_StraightTableReset.zip)

**Comment by Luis-Feilem** [comment]
Sorry for the delay, we got overflowed with work and I couldn't get a couple of hours to make this (priorities...).

Thanks you for your attention and support.
Cheers,
Luis.
[CustomReport_Issue_StraightTableReset.zip](https://github.com/ClimberAB/ClimberCustomReport/files/1058526/CustomReport_Issue_StraightTableReset.zip)

**Comment by Luis-Feilem** [comment]
By the way, I realized that both pivot and straight tables are reset on this particular custom report object. 

Luis.

**Comment by ClimberExtensions** [comment]
Hi Luis,
I've opened your app but can't recreate your problem. Could you please explain more in detail what you do to make the bug appear?
Pivot table is not supported in the Climber Custom Report at the moment so please explain what you are doing with the pivot table.
What version of Qlik Sense are you using and what version of the Climber Custom Report are you using?

**Comment by Luis-Feilem** [comment]
Good morning there,

I am uploading screen recordings of what I do exactly. Basically, I select a master table item, select the dimensions and measures I want the report to show, hit _Apply_ and then select a value from the table, to make a selection in its field. 
I... did not know that pivot table is not supported. However, I must say that I have used custom reports for pivot tables in other sheets and it worked wonderfully. I tagged _pivot_ the pivot master tables and then make a Custom Report based on that tag, so the reports are based off pivot tables. I have some sheets with custom reports based on pivot tables where this issue does not interfere, as opposed to straight tables custom reports :( 

Qlik Sense Desktop 3.1 SR6, CustomReport 2.0.2

Cheers,
Luis.
[CR_recordings.zip](https://github.com/ClimberAB/ClimberCustomReport/files/1060308/CR_recordings.zip)
